### PR TITLE
edit rn 13.1.1

### PR DESCRIPTION
**Pull request by marcofilippi**
## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensitive content has been committed

### CODE REVIEW

File: docs/release-notes/v13.1.1.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+The new version `2.6.5` of the Flow Manager is available!
+
+##### New Features
+
+- Added possibility to specify an optional JSON schema validation for incoming events payload
+- Added a setting to control whether or not unexpected events (i.e., events that are not mapped as outgoing transitions for the current saga state) should raise an error
+
+## How to update your Console
+
+For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `x.x.x`.
```
Comment (fredmaggiowski) [inline docs/release-notes/v13.1.1.md:66]: @mattia-fumo FYI

Comment (marcofilippi) [inline docs/release-notes/v13.1.1.md:66]:
```suggestion
For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v13.7.2-beta.0`.

```

File: docs/release-notes/v13.1.1.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- Added possibility to specify an optional JSON schema validation for incoming events payload
+- Added a setting to control whether or not unexpected events (i.e., events that are not mapped as outgoing transitions for the current saga state) should raise an error
+
+#### CRUD Service
+
+The new version `7.1.1` of the CRUD Service is available!
+
+##### Improvements
+
+* extend to `/bulk` endpoint the requests preprocessing performed on exposed routes of View with lookup feature enabled
```
Comment (albertotessarotto) [inline docs/release-notes/v13.1.1.md:104]:
```suggestion
* `/bulk` endpoints exposed by CRUD Views with lookup feature enabled are now working properly
```

File: docs/release-notes/v13.1.1.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+##### New Features
+
+- Added possibility to specify an optional JSON schema validation for incoming events payload
+- Added a setting to control whether or not unexpected events (i.e., events that are not mapped as outgoing transitions for the current saga state) should raise an error
+
+#### CRUD Service
+
+The new version `7.1.1` of the CRUD Service is available!
+
+##### Improvements
```
Comment (albertotessarotto) [inline docs/release-notes/v13.1.1.md:102]:
```suggestion
##### Bug Fix

```

File: docs/release-notes/v13.1.1.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+- Added possibility to specify an optional JSON schema validation for incoming events payload
+- Added a setting to control whether or not unexpected events (i.e., events that are not mapped as outgoing transitions for the current saga state) should raise an error
+
+#### CRUD Service
+
+The new version `7.1.1` of the CRUD Service is available!
+
+##### Bug Fix
+
+
```
Comment (albertotessarotto) [inline docs/release-notes/v13.1.1.md:104]:
```suggestion
```

File: docs/release-notes/v13.1.1.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+* `/bulk` endpoints exposed by CRUD Views with lookup feature enabled are now working properly
+
+## How to update your Console
+
+For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v13.7.2-beta.0`.
+
+
```
Comment (albertotessarotto) [inline docs/release-notes/v13.1.1.md:111]:
```suggestion
```

**Review by albertotessarotto — APPROVED** [review]

**Pull request merged**